content/docs/user-guide/interactivity/robotics/importing-robot.md

```diff
@@ -0,0 +1,115 @@
+
+![Robot Importer](/images/user-guide/gems/ros2/URDF_importer_fail.png)
+
+### Processing robot assets
+
+6. If the XACRO or URDF file is loaded correctly you will see a table with a list of all mesh files (assets) used in the model. Each row of the table describes one asset, with information divided into the following columns:
+
+- **URDF mesh path** - path extracted from the URDF
+- **Resolved mesh from URDF** - file path of the mesh located in user's filesystem
+- **Product asset** - location of the generated asset (`*.azmodel` or `*.pxmesh`) in the O3DE project assets directory
+- **Source asset** - location of the asset mesh file in the O3DE project assets directory
+- **Type** - the type of the asset, which can be `visual` or `collider`
+
+Besides this information, there is an indication if the asset was imported correctly (green) or if the import failed (red).
+
+Additional information about each asset can be obtained by double-clicking its row. This will open the O3DE Asset Processor showing import details. This information may be especially helpful in case of assets that were not imported correctly.
+
+Click **Next** when finished.
+
+![Robot Importer](/images/user-guide/gems/ros2/URDF_importer_mesh_list.png)
+
+**What to do if there are problems with importing assets?**
+If the Source asset field is marked as failed, it is caused by a URDF importer failure to find a reference to the mesh file in the file system. In such a situation, double-check whether the robot's description package was built and sourced correctly.
+If the Product asset field is marked as failed, it is caused by the Asset Builder failing to build the product asset. Refer to the Asset Processor output to resolve the issue.
+You can do either:
+
+- click **Back**, resolve problems with mesh files and click **Next** to retry import, or
+- proceed by clicking **Next** and finish URDF import. Missing meshes can be added afterward in the O3DE editor.
+
+### Creating a robot prefab
+
+7. On the next page:
+
+- Choose a prefab name
+- Indicate (using the checkbox) if robot physics should be based on articulations or classic rigid bodies and joints structure.
+
+Click **Create Prefab** when finished.
+
+![Robot Importer](/images/user-guide/gems/ros2/URDF_importer_prefab_creation.png)
+
+> Note: the decision whether to use articulations highly depends on the robot structure and simulation requirements. In general, articulations are more stable and computational efficient, but have some limitations. The 2 main limtations are:
+>
+>- Articulations are only available in PhysX 5. By default O3DE is build with PhysX 4.
+>- Articulations natively only support tree-structures - closed loops are not allowed.
+>
+>The "rule of thumb" would be to use articulations in robotic simulation whenever it is possible. However please refer to [this documentation](https://nvidia-omniverse.github.io/PhysX/physx/5.1.3/docs/Articulations.html) for more details.
```

Comment:
- Move to note callout box
- When linking to docs, use the name or description of the docs. If external docs, you must specify the source (who owns the docs)
```suggestion
    {{< note >}}
**Use articulation for joints and rigid bodies** depends on the robot structure and simulation requirements. In general, articulations are more stable and computational efficient, but have some limitations. The 2 main limtations are:
- Articulations are only available in PhysX 5. By default, O3DE is built with PhysX 4.
- Articulations only support tree-structures natively. Closed loops are not allowed.

We recommend to use articulations in robotic simulation whenever possible. For more details, refer to [Articulations](https://nvidia-omniverse.github.io/PhysX/physx/5.1.3/docs/Articulations.html) in NVIDIA PhysX 5 documentation.
    {{< /note >}}
```